src/client/components/AddResume/AddResume.js

```diff
@@ -0,0 +1,45 @@
+import React, {useState} from 'react';
+import ReactDom from 'react-dom';
+import Button from '../Button/Button';
+import './AddResume.css';
+import close from '../../assets/images/closeIcon.svg';
+
+export const AddResume = () => {
+const[title,setTitle] = useState('');
+const[description,setDescription] = useState('');
+const[uploadFile,setUploadaFile] =useState('');
+
+    return ReactDom.createPortal ((
+        <div className = "popup">
+            <div className = "popup-form">
+                <img className="closeicon" src= {close} alt='closeIcon'/>
```

Comment:
Human readable alt attribute

```suggestion
                <img className="closeicon" src= {close} alt='Close icon'/>
```